From f775c88d401ecad9d70063063c4adb8d5a988d34 Mon Sep 17 00:00:00 2001
From: Haseeb Ashraf <haseeb_ashraf@mentor.com>
Date: Fri, 1 Aug 2025 12:47:24 +0500
Subject: [PATCH] vulkan_enc/meson.build: remove TMPDIR reference

Using relative path for output_host_path. This path does not exist.
It seems like a porting issue from AOSP as the host/vulkan/cereal
directory exists there.

Since the path does not exist, the genvk.py script is not able to
convert it to relative path. Passing path which is already converted
to a relative path, fixes the QA warning for reference to TMPDIR.
```
WARNING: mesa-2_git-r0 do_package_qa: QA Issue:
File /usr/src/debug/mesa/git/src/gfxstream/guest/vulkan_enc/func_table.cpp in package mesa-src contains reference to TMPDIR
```

The path was appearing in autogenerated source files in code comments
which were adding command that was used to generate this source code.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Haseeb Ashraf <haseeb_ashraf@mentor.com>
---
 src/gfxstream/guest/vulkan_enc/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gfxstream/guest/vulkan_enc/meson.build b/src/gfxstream/guest/vulkan_enc/meson.build
index dd96a55cc9d..b592d902293 100644
--- a/src/gfxstream/guest/vulkan_enc/meson.build
+++ b/src/gfxstream/guest/vulkan_enc/meson.build
@@ -1,7 +1,7 @@
 # Copyright 2022 Android Open Source Project
 # SPDX-License-Identifier: MIT
 
-output_host_path = meson.current_build_dir() + '/host/vulkan/cereal'
+output_host_path = './host/vulkan/cereal'
 guest_enc_path = meson.current_build_dir()
 
 if build_machine.system() != 'android'
-- 
2.34.1

